OptimizelySDKShared/OptimizelySDKShared/OPTLYClient.m

```diff
@@ -133,13 +133,59 @@ - (BOOL)isFeatureEnabled:(nullable NSString *)featureKey userId:(nullable NSStri
     if (self.optimizely == nil) {
         [self.logger logMessage:OPTLYLoggerMessagesClientDummyOptimizelyError
                       withLevel:OptimizelyLogLevelError];
-        return nil;
+        return false;
     }
     else {
         return [self.optimizely isFeatureEnabled:featureKey userId:userId attributes:attributes];
     }
 }
 
+-(BOOL)getFeatureVariableBoolean:(NSString *)featureKey variableKey:(NSString *)variableKey userId:(NSString *)userId attributes:(NSDictionary<NSString *,NSString *> *)attributes {
```

Comment:
Line 143 and additional lines in OPTLYClient.m are too long.
Please line break long code lines like these so we can see all the code in GITHUB.COM diff.

Other OPTLYClient.m long lines: 150, 161, 166, 173, 178, 185 .